client/src/views/SignupCitizen.tsx

```diff
@@ -91,6 +160,7 @@ function SignupCitizen() {
 
   const handleSubmit = async (evt: React.FormEvent) => {
     evt.preventDefault();
+    handleNext();
```

Comment:
let's put this in the else below
and remove that history.push
that way we see the success page and aren't automatically move to the home screen